packages/cli/src/commands/bisect/index.ts

```diff
@@ -228,7 +223,8 @@ export default async function main(client: Client): Promise<number> {
     // If we have the "good" deployment in this chunk, then we're done
     for (let i = 0; i < newDeployments.length; i++) {
       if (newDeployments[i].url === good) {
-        newDeployments = newDeployments.slice(0, i + 1);
+        // grab all deployments up until the good one
+        newDeployments = newDeployments.slice(0, i);
```

Comment:
This appears to be a bug. In the code before, the "good" deployment was included in the list. This could make it be one of the deployments that `git bisect` asks you to check to see if it's good or bad, but you already told `git bisect` that you know it's good.